multiview/embed/splitae.py

```diff
@@ -0,0 +1,241 @@
+import torch
+from torch.utils.data import Dataset, DataLoader
+from torchvision import datasets
+import torchvision
+import matplotlib.pyplot as plt
+import PIL
+import numpy as np
+import itertools
+import tqdm
+
+from .base import BaseEmbed
+from ..utils.utils import check_Xs
+
+
+class _FullyConnectedNet(torch.nn.Module):
+    def __init__(self, inputSize, hiddenSize, numHiddenLayers, embeddingSize):
+        super().__init__()
+        assert numHiddenLayers >= 0, "can't have negative hidden layer count"
+        self.layers = torch.nn.ModuleList()
+        if numHiddenLayers == 0:
+            self.layers.append(torch.nn.Linear(inputSize, embeddingSize))
+        else:
+            self.layers.append(torch.nn.Linear(inputSize, hiddenSize))
+            for i in range(numHiddenLayers-1):
+                self.layers.append(torch.nn.Linear(hiddenSize, hiddenSize))
+            self.layers.append(torch.nn.Linear(hiddenSize, embeddingSize))
+
+    def forward(self, x):
```

Comment:
Same for all these functions: provide docs. They can be much less rigorous than public code.